UnityProject/Assets/Scripts/Shuttles/CargoShuttle.cs

```diff
@@ -169,10 +184,11 @@ private void UnloadCargo()
 				{
 					if (item.TryGetComponent<Attributes>(out var attributes))
 					{
-						if (attributes.ExportType == Attributes.CargoExportType.Never) continue;
+						// Items that cannot be sold in cargo will be ignored unless they have a trait that is assoicated with a bounty
+						if (attributes.CanBeSoldInCargo == false && hasBountyTrait(attributes) == false) continue;
 
 						// Don't sell secured objects e.g. conveyors.
-						if (attributes.ExportType != Attributes.CargoExportType.Always && behaviour.IsNotPushable) continue;
+						if (attributes.CanBeSoldInCargo && behaviour.IsNotPushable) continue;
```

Comment:
I think this should only be the pushable check?

Reply:
No because sometimes you can have something that can be sold but has been screwed which in that case it shouldn't be sold until it is unscrewed.

Reply:
If I'm reading correctly: If I have something that can't be sold normally but has a bounty and it's wrenched to the shuttle, it will still be sold.

You just want to prevent anything that's wrenched being sold, right?

Reply:
>You just want to prevent anything that's wrenched being sold, right?

Yes

Reply:
So you can just check `behaviour.IsNotPushable` and `continue` if it is. Maybe I'm misunderstanding?